src/main/resources/init_db.sql

```diff
@@ -1,13 +1,90 @@
+                                                       REFERENCES `internet_shop`.`roles` (`role_id`)
+                                                       ON DELETE NO ACTION
+                                                       ON UPDATE NO ACTION,
+                                               CONSTRAINT `users_fk`
+                                                   FOREIGN KEY (`user_id`)
+                                                       REFERENCES `internet_shop`.`users` (`user_id`)
+                                                       ON DELETE NO ACTION
+                                                       ON UPDATE NO ACTION);
+CREATE TABLE `internet_shop`.`shopping_carts` (
+                                                  `cart_id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                                                  `user_id` BIGINT(11) NOT NULL,
+                                                  `deleted` TINYINT(1) NOT NULL DEFAULT 0,
+                                                  PRIMARY KEY (`cart_id`),
+                                                  UNIQUE INDEX `user_id_UNIQUE` (`user_id` ASC) VISIBLE,
+                                                  CONSTRAINT `user_cart_fk`
+                                                      FOREIGN KEY (`user_id`)
+                                                          REFERENCES `internet_shop`.`users` (`user_id`)
+                                                          ON DELETE NO ACTION
+                                                          ON UPDATE NO ACTION);
+CREATE TABLE `internet_shop`.`orders_products` (
+                                                   `order_id` BIGINT(11) NOT NULL,
+                                                   `product_id` BIGINT(11) NOT NULL,
+                                                   INDEX `order_orders_products_fk_idx` (`order_id` ASC) VISIBLE,
+                                                   INDEX `product_orders_products_fk_idx` (`product_id` ASC) VISIBLE,
+                                                   CONSTRAINT `order_orders_products_fk`
+                                                       FOREIGN KEY (`order_id`)
+                                                           REFERENCES `internet_shop`.`orders` (`order_id`)
+                                                           ON DELETE NO ACTION
+                                                           ON UPDATE NO ACTION,
+                                                   CONSTRAINT `product_orders_products_fk`
+                                                       FOREIGN KEY (`product_id`)
+                                                           REFERENCES `internet_shop`.`products` (`product_id`)
+                                                           ON DELETE NO ACTION
+                                                           ON UPDATE NO ACTION);
```

Comment:
Let's add at least two lines of code needed to create the initial roles of your DB for the project to work properly